packages/fast-components-styles-msft/src/context-menu/index.ts

```diff
@@ -0,0 +1,29 @@
+import designSystemDefaults, {
+    DesignSystem,
+    withDesignSystemDefaults,
+} from "../design-system";
+import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
+import { ContextMenuClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { foregroundLarge } from "../utilities/colors";
+import { elevation, ElevationMultiplier } from "../utilities/elevation";
+
+const styles: ComponentStyles<ContextMenuClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<ContextMenuClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+
+    return {
+        contextMenu: {
+            ...elevation(ElevationMultiplier.e11, designSystem.foregroundColor)(
+                designSystem
+            ),
+            margin: "0",
+            padding: "4px 0",
+            maxWidth: "368px",
+            minWidth: "64px",
+            borderRadius: "2px",
+        },

```

Comment:
total nit: extra comma

Reply:
If this is important, prettier will remove. It may be adding it in as ES5 trailing comma.